src/Helmet.jsx

```diff
@@ -171,7 +171,7 @@ class Helmet extends React.Component {
     }
 
     static rewind() {
-        const title = serverTitle;
+        const title = HTMLEntities.encode(serverTitle);
```

Comment:
I was keen on doing 

```jsx
const title = `<title>${HTMLEntities.encode(serverTitle)}</title>`;
```

As the other two return the HTML, but that'd be a more breaking change.